app/overrides/add_sale_price_to_product_form.rb

```diff
@@ -0,0 +1,11 @@
+Deface::Override.new(
+  virtual_path: 'spree/admin/products/_form',
+  name: 'add_sale_price_to_product_form',
+  insert_after: "[data-hook='admin_product_form_price']",
+  text: "
+    <div class='form-group' data-hook='sale_price'>
+      <%= f.label :sale_price, raw(Spree.t(:sale_price) + content_tag(:span, ' *')) %>
+      <%= f.text_field :sale_price, value:
```

Comment:
@rishabhguptarishi as sale_price is a field in variant table, no need to have it shown in product forms